sites/cheerpj/src/content/docs/11-guides/implementing-native-methods.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Implementing Native Methods
+description: Java Native Interface (JNI) with CheerpJ
+---
+
+CheerpJ allows you to implement Java native methods (typically written in C/C++ or another AOT-compiled language) directly in JavaScript, similar to the Java Native Interface (JNI) in standard Java.
+
+In Java, native methods are identified by the `native` keyword in their declaration. These methods are not implemented in Java but are instead defined in an external language, which in the case of CheerpJ, is JavaScript.
+
+## Implementing Native Methods in JavaScript
+
+To implement a native method in JavaScript, create an `async` function that follows the naming convention `Java_<ClassName>_<MethodName>`. The function takes:
+
```

Comment:
We should add a part here that already shows how to add the native function in Java, maybe a header before the `Implementing Native Methods in JavaScript` and call it something like `How do add a native function to Java` and just show the same example class.

The old docs also had a part about the naming that was very clear, please add this:
`Java_<fully-qualified-class-name>_<method-name>` - For instance, if com.foo.Bar has a native method called `baz`, its object key is `Java_com_foo_Bar_baz`.